x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/prebuilt_rules/fleet_integration.ts

```diff
@@ -5,24 +5,23 @@
  * 2.0.
  */
 import expect from 'expect';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
```

Comment:
Yes, since the `FtrProviderContext` exported from `test_serverless/api_integration/ftr_provider_context` is actually `GenericFtrProviderContext`, my understanding is that is is a subset/used for both. The `services` passed to its generic parameter seem to be deployment independent + services specific to serverless, so both.

But anyways, I'll confirm this with Wafaa tomorrow. I see that DE also uses imports from different FtrProviderContext interchangably.

Reply:
👋  Wanted to jump here, to confirm our usage of the exported services from test_serverless/api_integration/ftr_provider_context. We utilize this because it loads the essential services necessary. In case a test in ESS requires additional services, we can extend it and include the necessary services specifically for that test.